README.md

```diff
@@ -767,169 +1206,248 @@ The description is an an optional string for a human-readable description of thi
 
 ##### `additional_attributes`
 
-Specifies any additional attributes to add to the Valve. Should be a hash of the format 'attribute' => 'value'. This parameter is optional.
+Specifies any additional attributes to add to the Valve. Should be a hash of the format 'attribute' => 'value'.
+
+Optional
 
 ##### `attributes_to_remove`
 
-Specifies any attributes to remove from the Valve. Should be a hash of the format 'attribute' => 'value'. This parameter is optional.
+Specifies any attributes to remove from the Valve. Should be a hash of the format 'attribute' => 'value'.
+
+Optional
 
 #### tomcat::install
 Installs the software into the given directory from a source Apache Tomcat tarball. Alternatively, it may be used to install a tomcat package.
 
 Tomcat instances may then be created from the install using `tomcat::instance` and pointing `tomcat::instance::catalina_home` to the directory managed by `tomcat::install`.
 
 ##### `catalina_home`
-specifies the directory of the Tomcat installation from which the instance should be created. Valid options: a string containing an absolute path. Default: $::tomcat::catalina_home.
+specifies the directory of the Tomcat installation from which the instance should be created.
+
+Valid options: a string containing an absolute path.
+
+Default value: `$::tomcat::catalina_home`.
 
 ##### `install_from_source`
 Specifies whether to install from source or from a package. If set to `true` installation uses the `source_url`, `source_strip_first_dir`, `user`, `group`, `manage_user`, and `manage_group` parameters. If set to `false` installation uses the `package_ensure`, `package_name`, and `package_options` parameters.
 
-Valid options: `true` and `false`. Default: `true`.
+Valid options: `true` and `false`.
+
+Default value: `true`.
 
 ##### `source_url`
 In single-instance mode: *Required if `install_from_source` is set to `true`.* Specifies the source URL to install from.
 
 Valid options: a string containing a `puppet://`, `http(s)://`, or `ftp://` URL.
 
 ##### `source_strip_first_dir`
-Specifies whether to strip the topmost directory of the tarball when unpacking it. Only valid if `install_from_source` is set to `true`. Valid options: `true` and `false`. Default: `true`.
+Specifies whether to strip the topmost directory of the tarball when unpacking it. Only valid if `install_from_source` is set to `true`.
+
+Valid options: `true` and `false`.
+
+Default value: `true`.
 
 ##### `environment`
 Environment variables for settings such as http_proxy, https_proxy, or ftp_proxy. These are passed through to the staging
 module and then to the underlying exec(s), so it follows the same format of the exec type `environment`
 https://docs.puppet.com/puppet/latest/reference/type.html#exec-attribute-environment
 
 ##### `user`
-Specifies the owner of the source installation directory. Default: `$::tomcat::user`
+Specifies the owner of the source installation directory.
+
+Default value: `$::tomcat::user`
```

Comment:
this (and the other default values like it) are strings, so should be in single quotes